src/app/functions/server/dashboard.ts

```diff
@@ -29,11 +29,14 @@ type Exposures = {
 
 type SanitizedExposures = Array<Record<string, number>>;
 export interface DashboardSummary {
-  dataBreachTotalNum: number;
-  dataBreachFixedNum: number;
-  dataBrokerTotalNum: number;
-  dataBrokerFixedNum: number;
-  dataBrokerInProgressNum: number;
+  dataBreachTotalNum: number; // total number of user data breaches
```

Comment:
I think if you add these comments as JSDoc comments, some editors like VSCode will display the descriptions when you hover over them at call sites?

```suggestion
  /** total number of user data breaches */
  dataBreachTotalNum: number;
```